locale/fr/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: index.hbs
+labels:
+  current-version: Version Actuelle
+  download: Téléchargements
+  download-for: Téléchargements pour
+  other-downloads: Autres Téléchargements
+  other-lts-downloads: Autres Téléchargements LTS
+  other-current-downloads: Autres Téléchargements Actuels
+  current: Actuelle
+  lts: LTS
+  tagline-current: Dernières Fonctionnalités
+  tagline-lts: Recommandé pour la plupart des utilisateurs
+  changelog: Journal des modifications
+  api: API Docs
+  version-schedule-prompt: Ou regardez le
+  version-schedule-prompt-link-text: Planning LTS
+---
+
+Node.js® est un environnement d'exécution JavaScript construit sur le [moteur JavaScript V8 de Chrome](https://developers.google.com/v8/).
+Node.js utilise un modèle basé sur l'événementiel et des entrées/sorties non bloquantes, ce qui le rend léger et efficace. L'écosystème de logiciels de Node.js, [npm](https://www.npmjs.com/), est le plus grand écosystème de librairies open source au monde.
```

Comment:
I'm agree with @harf18. "bibliothèques" is the correct form.
If we have a word to translate english in a easy way, we should use this.
"open source" can be translate to "code source ouvert" but in some cases this translation is bad.
**To put it in a nutshell "open source"≠"code source ouvert" but "librairies" = "bibliothèques"**.